tests/lint/actions_ci.py

```diff
@@ -23,7 +23,7 @@ def test_actions_ci_fail_wrong_nf(self):
     self.lint_obj._load()
     self.lint_obj.minNextflowVersion = "1.2.3"
     results = self.lint_obj.actions_ci()
-    assert results["failed"] == ["Minimum NF version in '.github/workflows/ci.yml' different to pipeline's manifest"]
+    assert results["failed"] == ["Minimum pipeline NF version '1.2.3' is not tested in '.github/workflows/ci.yml'"]
```

Comment:
```suggestion
    assert results["failed"] == ["Minimum pipeline NF version 'YY.MM.n' is not tested in '.github/workflows/ci.yml'"]
```

Reply:
I don't even know what that means before and after the suggestion! 😆

Reply:
Ah, I think it is a direct replacement for the version specified above because it will be eventually replaced in reality anyway:

```
self.lint_obj.minNextflowVersion = "1.2.3"
```

Reply:
Ah, ok!